rosbag2_cpp/test/rosbag2_cpp/test_sequential_writer.cpp

```diff
@@ -47,6 +47,9 @@ class SequentialWriterTest : public Test
     storage_options_ = rosbag2_cpp::StorageOptions{};
     storage_options_.uri = "uri";
 
+    rcpputils::fs::path dir(storage_options_.uri);
+    rcpputils::fs::remove(dir);
```

Comment:
I actually think that this should be `remove_all` in case the directory is not empty.